examples/blame_example.ipynb

```diff
@@ -67,212 +61,246 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
-   "metadata": {},
+   "execution_count": null,
+   "metadata": {
+    "id": "kT3m0-zoNdvW"
+   },
    "outputs": [],
    "source": [
     "location_info_response = iai.location_info(location=location)\n",
     "rendered_static_map = location_info_response.birdview_image.decode()\n",
-    "scene_plotter = iai.utils.ScenePlotter(rendered_static_map, \n",
-    "                                       location_info_response.map_fov, \n",
-    "                                       (location_info_response.map_center.x, location_info_response.map_center.y), \n",
+    "scene_plotter = iai.utils.ScenePlotter(rendered_static_map,\n",
+    "                                       location_info_response.map_fov,\n",
+    "                                       (location_info_response.map_center.x, location_info_response.map_center.y),\n",
     "                                       location_info_response.static_actors)"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
-   "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "[((0, 3), (89, 96), (0,)), ((2, 17), (135, 150), (2,)), ((3, 5), (150, 162), (3,)), ((4, 11), (1, 15), (11,)), ((4, 21), (61, 61), (21,)), ((11, 21), (16, 29), (21,)), ((12, 17), (193, 196), (12,)), ((17, 22), (120, 121), (22,))]\n"
-     ]
-    }
-   ],
+   "execution_count": null,
+   "metadata": {
+    "id": "L0ufxMO8NdvX",
+    "scrolled": true
+   },
+   "outputs": [],
    "source": [
-    "collisions = None\n",
+    "@dataclass\n",
+    "class LogCollision:\n",
+    "    collision_agents: Tuple[int, int]\n",
+    "    start_time: int \n",
+    "    end_time: int\n",
+    "\n",
+    "def compute_pairwise_collisions(agent_states_history,agent_attributes):\n",
```

Comment:
It would be nice if you can wrap some code blocks in this function into new functions and try using meaningful variable/function names. In this way, the code can be self-explanatory and we don't need too many inline comments.

Reply:
Any code blocks that do not reference above-scope variables have been placed into their own functions for clarity.